developer-docs-site/docs/guides/transaction-management.md

```diff
@@ -0,0 +1,97 @@
+# Transaction Management
+
+This guide explains how to build a transaction management harness that can scale on the Aptos blockchain.
+
+## Building a Transaction Manager
+
+This section covers the major aspects of building a transaction management harness. This consists of the following core components:
+
+- A sequence number generator that allocates and manages available sequence numbers for a single account.
+- A transaction manager that receives payloads from an application or a user, sequence numbers from the sequence number generator, and has access to the account key to combine the three pieces together into a viable signed transaction. It then also takes the responsibility for pushing the transaction to the blockchain.
+- An on-chain worker, leader harness that let’s multiple accounts share the signer of a single shared account.
+
+Currently this framework assumes that the network builds no substantial queue, that is a transaction that is submitted executes and commits with little to no delay. In order to address high demand, this work needs to be extended with the following components:
+
+- Optimizing `base_gas_unit` price to ensure priority transactions can be committed to the blockchain.
+- Further handling of transaction processing rates to ensure that the expiration timer is properly set.
```

Comment:
add:

It is expected a layer on top to be handling retries, for each individual transaction, and creating new (same or different payloads) 

Due to nature of sequence numbers, it is required that each account is managed only by a single machine/instance of transaction manager. Otherwise each instance will be creating transactions with overlapping sequence numbers, and in-memory state of Transaction Manager will become stale as well.